plugins/mcreator-localization/help/default/entity/data_default_value.md

```diff
@@ -0,0 +1,4 @@
+This is the default value of the entity's variable.
+- **Number**: It can go from the maximal negative limit to the maximal positive limit of an integer.
+- **Logic**: true or false
+- **String**: A default text
```

Comment:
How can text be default?
```suggestion
- **Number**: Can go from the maximal negative limit to the maximal positive limit of an integer;
- **Logic**: _true_ or _false_;
- **String**: Simple text.
```

Reply:
> How can text be default?

I didn't know what to write for this one. xD

Reply:
Keep the original order, so they are ordered the same way as inside the list.

Reply:
I thought this is like the default order of primitive types, similar to what was in procedures when creating a local variable.

Reply:
Increasing in "complexity": boolean -> 2 values, integer -> 16^8 values, string -> ≈∞ values.